src/report/js/ReportBuilder.js

```diff
@@ -3548,22 +3548,45 @@ define([
                 }, (err) => {
                   document.getElementById('firesLineChartLoading').remove();
                 });
-
-                let totalFireAlertsUrl;
+                
+              // Old query to get the joined alerts of Modis and Viirs for a specific region 
+                // let totalFireAlertsUrl;
+                // if (window.reportOptions.aoiId) {
+                //   totalFireAlertsUrl = Config.fires_api_endpoint + 'admin/' + queryFor + '/' + window.reportOptions.aoiId + '?period=' + self.startDateRaw + ',' + self.endDateRaw;
+                // } else {
+                //   totalFireAlertsUrl = Config.fires_api_endpoint + 'admin/' + queryFor + '?period=' + self.startDateRaw + ',' + self.endDateRaw;
+                // }
+
+                // request.get(totalFireAlertsUrl, {
+                //   handleAs: 'json'
+                // }).then((res) => {
+                //   const total = res.data.attributes.value[0].alerts;
+
+                //   // $("#totalFireAlerts").html(self.numberWithCommas(total));
+                // });
 
```

Comment:
We used to query Viirs and Modis and sum their fires to get the total. Now, we're displaying each total in the header of the Reports tab.